vortex-serde/src/layouts/read/recordbatchreader.rs

```diff
@@ -21,36 +21,55 @@ fn vortex_to_arrow(result: VortexResult<Array>) -> Result<RecordBatch, ArrowErro
         .map_err(vortex_to_arrow_error)
 }
 
-pub struct VortexRecordBatchReader<'a, R: VortexReadAt + Unpin + 'static> {
+pub trait Spawn {
```

Comment:
Spawn is kind of a weird name for this trait, since it doesn't spawn things... it just has a blocks_on fn. maybe Join?

Reply:
How about just calling it `AsyncRuntime`

Reply:
@a10y wins